replica_aggregator/src/main/scala/uk/ac/wellcome/platform/storage/replica_aggregator/services/ReplicaAggregatorWorker.scala

```diff
@@ -69,6 +69,17 @@ class ReplicaAggregatorWorker[IngestDestination, OutgoingDestination](
     val startTime = Instant.now()
 
     val ingestStep = getKnownReplicas(payload.replicaResult) match {
+      case Left(AggregationFailure(err: RetryableError)) =>
```

Comment:
This is wrong, I think? That’s an UpdateError; I want to catch an UpdateError that's wrapping a RetryableError.